### ISSUE TITLE
Duplicate aliases entries on JAR

### ISSUE THREAD

**Issue by TheDGOfficial** [SkriptLang/Skript#3727]
### Description
I see duplicate aliases entries on the latest (v2.5.3) Skript.jar file when downloaded from the releases page.

### Steps to Reproduce
Download the latest (v2.5.3) Skript.jar file from the Releases page and open it with any archive editor, i.e WinRAR, then navigate to the aliases-english directory.

Note: It seems some archive editors/programs/OS native ZIP/JAR/RAR openers does not show duplicate entries. See my last comment for a Java file to print the duplicate entries.

### Expected Behavior
There should be no duplicate entries.

### Errors / Screenshots
![image](https://user-images.githubusercontent.com/24778409/106397733-e42f4900-641f-11eb-8da0-a8833c8a5109.png)

Everything there is duplicated, including the README.md file. However, the classes and the README.md file on the root folder is not duplicated, and so this probably a problem of skript-aliases, but I see no duplicate files on the SkriptLang/skript-aliases repository, so this must be an error on gradle config while preparing the JAR file or something, so this issue belongs to this repository.

### Server Information
* **Server version/platform:** any
* **Skript version:** 2.5.3

### Additional Context

I tested with 2.5.2 and 2.5.1 too. The 2.5.1 does not contain these duplicate entries, so it started happening after 2.5.2.

List of changes between 2.5.1 and 2.5.2 can be viewed here:
https://github.com/SkriptLang/Skript/compare/2.5.1...2.5.2

I think it is because of the changes in build.gradle, but I'm not sure which one.

**Comment by ShaneBeee** [comment]
I just downloaded the 2.5.3 jar, and I could not replicate the issue

<img width="1081" alt="Screen Shot 2021-01-31 at 1 08 58 PM" src="https://user-images.githubusercontent.com/20008482/106398055-9685f280-63c5-11eb-9326-2e4f3beba427.png">

I also opened the jar up with Luyten (same thing as JD-GUI)
Same result, no doubles here.
<img width="1578" alt="Screen Shot 2021-01-31 at 1 10 56 PM" src="https://user-images.githubusercontent.com/20008482/106398087-d1882600-63c5-11eb-9596-8742548d269b.png">

**Comment by TheDGOfficial** [comment]
The JD-GUI does not list duplicate entries on my computer, too. I think it just does not display them. But opening the archive with WinRAR or programmatically with Java, the duplicate files are there, and this does not happen with < 2.5.1.

```
Exception in thread "main" java.util.zip.ZipException: duplicate entry: aliases-english/z-categories.sk
        at java.base/java.util.zip.ZipOutputStream.putNextEntry(ZipOutputStream.java:232)
```

**Comment by Romitou** [comment]
I can replicate this issue.
![image](https://user-images.githubusercontent.com/35146758/106398401-da531900-6412-11eb-9c57-25355acf38fd.png)

**Comment by TheDGOfficial** [comment]
Here is a quick Java file to print JarEntry objects in the JarFile that contains aliases-english.

```java
import java.util.jar.JarFile;
import java.io.IOException;

final class Main {
	public static final void main(final String[] args) {
		try {
			new JarFile("Skript.jar").stream().forEach(jarEntry -> {
				if (jarEntry.getName().contains("aliases-english")) {
					System.out.println(jarEntry.getName());
				}
			});
		} catch (final IOException e) {
			e.printStackTrace();
		}			
	}
}
```

The result of the execution (put the file into the same location as Skript.jar and run with java filename.java, only works on newer JDKs):

![image](https://user-images.githubusercontent.com/24778409/106398616-ec3db780-6424-11eb-84fa-d2420f15d50e.png)

It starts printing duplicate entries after the last file, +global-variations.sk

If you have a older JDK you might need to compile it first and package as a JAR.

**Comment by TPGamesNL** [comment]
Can confirm. Results per Gradle task:
Task `build`:
- `Skript.jar`: normal (expected) result, aliases-english present without duplicate file names.
- `Skript-*-nightly-*.jar`: no aliases-english directory.

Task `nightlyRelease`: 
- files in aliases-english doubled

Task `githubRelease`:
- files in aliases-english doubled.

**Comment by TPGamesNL** [comment]
Fixed in #3768
